Hspi/html/feature.js

```diff
@@ -0,0 +1,40 @@
+function getUrlParameterOrEmpty(key) {
+	return getUrlParameter(key) ?? '';
+};
+
+function getUrlParameter(sParam) {
+    var sPageURL = window.location.search.substring(1),
+        sURLVariables = sPageURL.split('&'),
+        sParameterName,
+        i;
+
+    for (i = 0; i < sURLVariables.length; i++) {
+        sParameterName = sURLVariables[i].split('=');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/dk307/HSPI_InfluxDBPersistence/pullRequest?prid=8479377)
```suggestion
        sParameterName = sURLVariables[i].split("=");
```